cmd/setup.go

```diff
@@ -110,8 +110,12 @@ func runSetupError(cmd *cobra.Command, args []string) error {
 	pa := patterns.New(logger)
 	w := wait.New(logger, d, pa)
 	pCfg := &project.Config{
-		Name: projectName,
-		Tag:  projectTag,
+		Name:       projectName,
+		K8sContext: "minikube",
+		Tag:        projectTag,
+	}
+	if existingCluster {
+		pCfg.K8sContext = "giantswarm-e2e"
 	}
```

Comment:
Staring new discussion: I think people rely on that when doing testing on local machine. I think it would be good to add a flag like `e2e-harness setup --kubecontext ...` and it should default to `minikube`. Then it's fully backward compatible.

Reply:
But before doing effort here I'd wait for others opinions. I'm far from being an expert here.

Reply:
I was thinking about it too, but the thing is that the name of context is decided inside of the code.
Its either `minikube` when minikube is spawned or `giantswarm-e2e` when existing cluster is used ( there is template frot he kubeconfig in the e2e-harness code) so user from outside doesnt have any real option ( unless he change code)

Reply:
> thing is that the name of context is decided inside of the code

Yes it was just hardcoded. But when you make it a flag and default to `minikube` this is backward compatible, and doesn't mess local testing.